### DISCUSSION TITLE
ダークモードの記事の色追従が若干遅れる

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#460] [要望・アイデア・不具合]
遅ればせながら、ダークモードを試してみました。
記事を表示した一瞬は従来と同じ色で表示され、0.3秒程度？で完全にダークモードの色になります。黒と白が頻繁に入れ替わりチカチカして目の負荷が大きく、非ダークモードよりつらみを感じました。
Windows10+Google Chrome 112.0.5615.50で確認しました。

![ダークモード](https://user-images.githubusercontent.com/12453432/230275417-95d78424-db25-474f-8d0d-5e4ae45c9e7f.jpg)

**Comment by yoshi389111** [comment]
通りすがりの者ですが、以前ツイッターの Qiita 公式アカウントで、ベータ版での制限だ、みたいな話をしていました。

https://twitter.com/Qiita/status/1635560899628130304

> ダークモードで表示する際の画面のちらつきについては、ダークモードを全ページに適用した際に解消すると思われるため、もうしばらくご容赦いただけますと幸いです🙇

（通常のスタイルシートを適用したあとに、ベータ版のスタイルを適用するからちらつくんですかね）

**Reply by radian-jp** [reply]
なるほど、ベータ版のうちはこのままって事ですかね。

**Reply by tanaka-asami415** [reply]
@yoshi389111 
代わりにご回答いただきありがとうございます🙏
> （通常のスタイルシートを適用したあとに、ベータ版のスタイルを適用するからちらつくんですかね）

（概ねそうです…）

@radian-jp 
申し訳ないです…🙏
ベータ版のうちはご不便をおかけしてしまいますが、ご容赦いただけますと幸いです🙇

**Comment by tanaka-asami415** [comment]
@radian-jp @yoshi389111 

ダークモードで表示する際のちらつきを一部解消しました！
https://qiita.com/release-notes/2023-05-23

完全な解消は、ベータ版から全体に適用した際に実現できると思われるため、もうしばらくご容赦いただけますと幸いです🙏

**Comment by tanaka-asami415** [comment]
@radian-jp 
ご報告が遅くなり申し訳ありません🙏

本リリースの前に、ダークモード（ベータ版）のちらつきを全ページで解消いたしました！
https://twitter.com/Qiita/status/1690983851781480448?s=20

引き続き正式リリースに向けて調整を行っておりますので、
お気づきの点などございましたら、フィードバックを頂ければ幸いです！✨
